devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -221,19 +221,41 @@ def refresh() {
 }
 
 def installed() {
-	if (((device.getDataValue("manufacturer") == "MRVL") && (device.getDataValue("model") == "MZ100")) || (device.getDataValue("manufacturer") == "OSRAM SYLVANIA") || (device.getDataValue("manufacturer") == "OSRAM")) {
+	if ((isMRVL() && (device.getDataValue("model") == "MZ100")) || isOsram() || isOsramSylvania()) {
 		if ((device.currentState("level")?.value == null) || (device.currentState("level")?.value == 0)) {
 			sendEvent(name: "level", value: 100)
 		}
 	}
 }
 
+def isLeviton() {
+	device.getDataValue("manufacturer") == "Leviton"
+}
+
+def isMRVL() {
+	device.getDataValue("manufacturer") == "MRVL"
+}
+
+def isOsram() {
+	device.getDataValue("manufacturer") == "OSRAM"
+}
+
+def isOsramSylvania() {
+	device.getDataValue("manufacturer") == "OSRAM SYLVANIA"
+}
+
+def isSengled() {
+	device.getDataValue("manufacturer") == "sengled"
+}
+
 def configure() {
 	log.debug "Configuring Reporting and Bindings."
 	// Device-Watch allows 2 check-in misses from device + ping (plus 1 min lag time)
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
 	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
-	refresh() + zigbee.onOffConfig(0, 300) + zigbee.levelConfig()
+	def cmds = refresh() + zigbee.onOffConfig(0, 300)
+	cmds += isLeviton() ? zigbee.levelConfig(1,3600, 0x00) : zigbee.levelConfig()
```

Comment:
I'm not sure this different reporting config will do anything.  It already has a reportable change of 1, and a value of 0 is not functionally different than 1.

Reply:
I removed that part of the code.